mesonbuild/interpreter.py

```diff
@@ -2211,7 +2225,16 @@ def func_subproject(self, nodes, args, kwargs):
         dirname = args[0]
         return self.do_subproject(dirname, kwargs)
 
+    def disabled_subproject(self, dirname):
+        self.subprojects[dirname] = SubprojectHolder(None, self.subproject_dir, dirname)
+        return self.subprojects[dirname]
+
     def do_subproject(self, dirname, kwargs):
+        disabled, required, feature = extract_required_kwarg(kwargs)
```

Comment:
The CI is right actually, you're missing an argument here.

Reply:
How so?

```
def extract_required_kwarg(kwargs):
```

And this is L2233 not `  File "/root/mesonbuild/interpreter.py", line 2239, in do_subproject`

Reply:
Rebase on master, here I have `def extract_required_kwarg(kwargs, subproject):`

Reply:
oO that means the CI system rebases for use... that sounds terrible! Thanks anyway :-)

Reply:
Didn't know that neither, but it actually make sense, since it's a merge request, better test that the merge works.

Reply:
yeah, or kindly ask you to rebase if necessary (gitlab does that afaics), both make sense to me indeed.